content/en/logs/log_configuration/attributes_naming_convention.md

```diff
@@ -83,7 +83,7 @@ The standard attribute panel appears when you add a new standard attribute or ed
 
 ### Default standard attribute list
 
-The default standard attribute list is split into the functional domains:
+See the full list of [Log Management Default Standard Attributes][9], which is split into the functional domains:
 
 - [Network/communications](#web-access)
```

Comment:
@maycmlee We need to update all these shortlinks as well.
`#web-access` should redirect instead to `https://docs-staging.datadoghq.com/may/update-logs-standard-attibutes/standard-attributes/?product=log+management&search=network.` (notice the search query done to filter by prefix)

Reply:
@LucasC good catch. I added the links and also pulled back in some of the information we had for each section. Let me know know what you think, thanks!